src/test/java/by/diploma/gym/service/GymInfoServiceImplTest.java

```diff
@@ -0,0 +1,120 @@
+package by.diploma.gym.service;
+
+import by.diploma.gym.dto.request.GymInfoRequest;
+import by.diploma.gym.dto.response.GymInfoResponse;
+import by.diploma.gym.mapper.GymInfoMapper;
+import by.diploma.gym.model.GymInfo;
+import by.diploma.gym.repository.GymInfoRepository;
+import by.diploma.gym.service.Impl.GymInfoServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class GymInfoServiceImplTest {
+
+    @InjectMocks
+    private GymInfoServiceImpl gymInfoService;
+
+    @Mock
+    private GymInfoRepository gymInfoRepository;
+
+    @Mock
+    private GymInfoMapper gymInfoMapper;
+
+    private final UUID id = UUID.randomUUID();
+    private final String address = "Main St";
+
+    private final GymInfoRequest request = new GymInfoRequest();
+    private final GymInfo entity = new GymInfo();
+    private final GymInfoResponse response = new GymInfoResponse();
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+
+        request.setAddress(address);
+        request.setPhoneNumber("123 456789");
+        request.setDescription("Nice gym");
+
+        entity.setId(id);
+        entity.setAddress(address);
+
+        response.setId(id);
+        response.setAddress(address);
+    }
+
+    @Test
+    void testCreate() {
```

Comment:
pls use more informative test name, f.e.:
test_<method_name>_should_<what we tests>